SINGE_Example.m

```diff
@@ -0,0 +1,47 @@
+%% Simple example that runs SCINGE for two replicates of two hyperparameter settings
```

Comment:
How much of this file changed?  If you delete the old `SCINGE_Example.m` in the same commit where you add this file, git may recognize that you renamed the file.  That would make comparing the new and old version easier.

Please convert the name here as well